packages/examples/packages/ledger/README.md

```diff
@@ -0,0 +1,30 @@
+# `@metamask/lifecylce-hooks-example-snap`
```

Comment:
Is this how it is spelled in the README that this was copied from? 😂

Reply:
Yes 😅